frontend/public/components/_catalog-item.scss

```diff
@@ -66,3 +66,26 @@ a.co-catalog-item-tile {
     text-decoration: none;
   }
 }
+
+.co-catalog-page {
```

Comment:
We should be following BEM naming here:

* co-catalog-page
* co-catalog-page__tabs
* co-catalog-page__content
* co-catalog-page__heading
* co-catalog-page__num-items